examples/camera_streamer/operators/oakd_camera/oakd_camera_op.py

```
@@ -0,0 +1,596 @@
+                    self.metadata["frame_timestamp_us"] = self._extract_timestamp_us(
+                        frame_right
+                    )
+                    self.metadata["stream_id"] = self._right_stream_id
+                    self.metadata["sequence"] = self._frame_count
+                    op_output.emit(
+                        as_tensor(right_data),
+                        "right_frame",
+                        emitter_name="holoscan::Tensor",
+                    )
+            except Exception:
+                pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)